packages/tokens/src/Tokens.story.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '.';
+
+const gutter = css`
+  margin-left: ${spacing[3]};
+`;
+
+const textWrapper = css`
+  margin-bottom: ${spacing[2]};
+  font-size: 10px;
+`;
+
+const spacingBlockVariants = {
+  1: css`
+    background-color: #fb4949;
+    width: ${spacing[1]};
```

Comment:
I wish we could do `spacing.1`

If anything having `spacing-1` as a value is cool, but dunno how we'd do that.

Reply:
me too re `.1` we can do `tokens.spacing-1`

Reply:
:eyes:

Not sure if this is actually better, but I'm awake for some reason so here's a thing 😁 

```tsx
const colors = [
  '#fb4949',
  '#497ffb',
  '#62e3fd',
  '#52c825',
  '#fdd063',
  '#fd7fec',
  '#a5fd8b',
];

const spacingBlockVariants = Object.keys(spacing).reduce((acc, key, idx) => {
  acc[key] = css`
    background-color: ${colors[idx]};
    width: spacing[key];
    height: spacing[key];
  `;
  return acc;
}, {});
```

Reply:
👋 @jetpacmonkey i dig this